docs/conf.py

```diff
@@ -126,6 +127,20 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
+version_or_name = subprocess.run(
+    "git symbolic-ref -q --short HEAD || git describe --tags --exact-match",
+    capture_output=True,
+    shell=True,
+    text=True,
+).stdout.strip()
+
+env = os.environ
+json_url = (
+    "https://deltares.github.io/imod-python/_static/switcher.json"
+    if "JSON_URL" not in env
+    else env["JSON_URL"]
```

Comment:
Please add a comment when JSON_URL is added to the environment when running multidoc.py